service_mesh/service_mesh_day_two/prepare-to-deploy-applications-ossm.adoc

```diff
@@ -22,6 +22,15 @@ include::modules/ossm-automatic-sidecar-injection.adoc[leveloffset=+2]
 
 include::modules/ossm-mixer-policy.adoc[leveloffset=+1]
 
+[id="ossm-deploy-set-correct-np_{context}"]
+== Setting the correct network policy
+
+{ProductShortName} creates network policies in the control plane and member namespaces to whitelist traffic between them. Before you deploy, consider the following conditions to ensure the services in your mesh that were previously exposed through an {product-title} route.
+
+* Traffic into the mesh must always go through the ingress-gateway for Istio to work properly.
+* Deploy services external to the mesh in separate namespaces that are not in any mesh.
+* If services that are outside the mesh must reside in a namespace that is part of a mesh, label the pods with `maistra.io/expose-route: "true"`, which ensures {product-title} routes to these services still work.
+
```

Comment:
lgtm. @knrc can you take a look?

Reply:
Non-mesh services which need to be deployed within a service mesh enlisted namespace should label their `Deployments` with ....